spec/interactors/knight_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Knight do
+  describe '::call' do
```

Comment:
What does the "::" in "::call" indicate? Yeah, yeah, I know that **I** technically wrote this, haha.